test/functional/test_framework/test_node.py

```diff
@@ -324,13 +325,13 @@ def is_node_stopped(self):
         return True
 
     def wait_until_stopped(self, timeout=BITCOIND_PROC_WAIT_TIMEOUT):
-        wait_until(self.is_node_stopped, timeout=timeout)
+        wait_until(self.is_node_stopped, timeout=timeout, lock=None, factor=self.factor)
```

Comment:
```suggestion
        wait_until(self.is_node_stopped, timeout=timeout, factor=self.factor)
```

None is the default value already